website/README.md

```diff
@@ -297,36 +324,95 @@ Sometimes you may want to include a category but not have a need for an index pa
+[
+  {
+    "title": "Indexless Category",
+    "routes": [
+      {
+        "title": "File",
+        "path": "indexless-category/file"
+      }
+    ]
+  }
+]
 ```
 
-The addition of the `name` property to a category object is all it takes to be able to skip the index file.
-
 ### Custom or External Links
 
 Sometimes you may have a need to include a link that is not directly to a file within the docs hierarchy. This can also be supported using a different pattern. For example:
 
-```js
-export default {
-  category: 'directory',
-  content: [
-    'file',
-    'another-file',
-    { title: 'Tao of HashiCorp', href: 'https://www.hashicorp.com/tao-of-hashicorp' }
-    }
-  ]
-}
+```json
+[
+  {
+    "name": "Directory",
+    "routes": [
+      {
+        "title": "File",
+        "path": "directory/file"
+      },
+      {
+        "title": "Another File",
+        "path": "directory/another-file"
+      },
+      {
+        "title": "Tao of HashiCorp",
+        "href": "https://www.hashicorp.com/tao-of-hashicorp"
+      }
+    ]
+  }
+]
 ```
 
 If the link provided in the `href` property is external, it will display a small icon indicating this. If it's internal, it will appear the same way as any other direct file link.
 
-<!-- END: editing-docs-sidebars -->
+### Plugin Docs
+
+Plugin documentation may be located within the `packer` repository, or split out into separate `packer-plugin-` repositories. For plugin docs within the `packer` repository, the process for authoring files and managing sidebar data is identical to the process for other documentation.
+
+For plugins in separate repositories, additional configuration is required.
+
+#### Setting up remote plugin docs
+
+Some setup is required to include docs from remote plugin repositories on the [packer.io/docs](https://www.packer.io/docs) site.
+
+1. The plugin repository needs to include a `docs.zip` asset in its release
+2. The `packer` repository must have a corresponding entry in `website/data/docs-remote-plugins.json` which points to the plugin repository.
+
+The `docs.zip` release asset is expected to be generated as part of the standard release process for `packer-plugin-*` repositories. Additional details on this process can be found in [the `packer-plugin-scaffolding` `README`](https://github.com/hashicorp/packer-plugin-scaffolding#registering-documentation-on-packerio).
+
+The `docs-remote-plugins.json` file contains an array of entries. Each entry points to a plugin repository. The `{ title, path, repo, version }` properties are required for each entry.
+
+```json5
+[
+  {
+    // ALL FIELDS ARE REQUIRED.
+    // "title" sets the human-readable title shown in navigation
+    title: 'Scaffolding',
+    // "path" sets the URL subpath under the component URL (eg `docs/builders`)
+    path: 'scaffolding',
+    // "repo" points to the plugin repo, in the format "organization/repo-name"
+    // if the organization == hashicorp, the plugin docs will be labelled "official".
```

Comment:
I remember a conversation about making partner's plugins official as well but I couldn't find the conversation and the result of it. In case that came to happen in the future, I suppose this will be easy to change, right?

Reply:
@sylviamoss great callout, yeah this should be relatively easy to adjust in the future - eg we could make the `official` status directly configurable in `docs-remote-plugins.json`, and fallback to inferring it from the org 👍

Reply:
Nice! Thank you!